.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ jobs:
   build:
     working_directory: /mnt/ramdisk/project
     docker:
-      - image: cimg/node:lts
+      - image: cimg/node:16.17
```

Comment:
downgraded due to https://app.circleci.com/pipelines/github/stoplightio/elements/8184/workflows/10807cca-a884-425f-893f-4a51bc7701c8/jobs/34378.
We'll need to address it at some point